src/main/java/de/jcm/discordgamesdk/Core.java

```diff
@@ -444,6 +444,13 @@ public void runCallbacks()
 		}
 	}
 
+	public void isDiscordRunning() {
```

Comment:
Making the method static is complicated, since the `isAvailable()` method of the channel also checks, whether the channel is still open or not, so it requires an open channel to work correctly.

Right now the channel itself is only opened when a new core is instantiated. I could also open a channel just to check, whether it could be opened or not, but this seem like a bit too much overhead to me.

Another option would be, to remove the check for openness and only check whether the socket/pipe-file exists, but then the method might miss some cases where the file still exists, but the channel is unusable for other reasons.

Therefore I would prefer the non-static approach. What do you think?

Reply:
Yes, it seems to be the most reasonable solution to keep it non-static.

I think it's overall not really possible to reliably predict if Discord is available before creating a Core anyway.